mindera-people/Sources/Shared/DTO/Extensions/EventDTO.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import MinderaPeopleKMMLib
+
+extension EventDTO: SwiftConvertible {
+    public typealias SwiftDTO = Event
+    
+    public func toSwift() throws -> Event {
+        var project: Project? = nil
+        
+        guard let id = UUID(uuidString: self.id) else {
+            throw ConversionError.invalidUUIDString
+        }
+        
+        if let optionalproject = self.project {
+            project = try optionalproject.toSwift()
+        }
+        
+        return Event(
+            id: id,
+            policy: try policy.toSwift(),
+            startDate: try Date().fromString(startDate),
+            endDate: try Date().fromString(endDate),
+            partOfDay: try partOfDay.toSwift(),
+            additionalInfo: additionalInfo,
+            includesBreakfast: includesBreakfast?.asBool(),
+            city: city,
+            project: project
```

Comment:
```suggestion
            project = project?.toSwift()
```

Event.project is an optional so you can put directly the method instead trying to unwrap 
You can remove:

> var project: Project? = nil
> if let optionalproject = self.project {
>             project = try optionalproject.toSwift()
>         }